OverlayPlugin.Core/EventSources/MiniParseEventSource.cs

```diff
@@ -547,27 +550,21 @@ protected override void Update()
                 DispatchEvent(this.CreateCombatData());
             }
 
-            if (importing && HasSubscriber(ImportedLogLinesEvent))
+            if (HasSubscriber(ImportedLogLinesEvent))
             {
-                List<string> logs = null;
-
                 lock (importedLogs)
                 {
                     if (importedLogs.Count > 0)
                     {
-                        logs = importedLogs;
+                        DispatchEvent(JObject.FromObject(new
```

Comment:
Move DispatchEvent into the lock context can cause `importedLogs` lock be taken until event dispatched. Is this intended?